app/assets/stylesheets/application.scss

```diff
@@ -9,3 +9,11 @@ $success-green: #00823b;
 @import "../node_modules/govuk-frontend/all";
 @import "bar";
 @import "success_summary";
+
+.access-request-preview-table {
+  max-width: 38em;
```

Comment:
@dankmitchell this isn't in its own partial because I figure maybe there's already something in the design system to do it, so didn't bother refactoring, is there something cleaner that you know of?

Reply:
I'm fine with this 👌

Reply:
You could wrap the table in a column and then remove this.

```
<div class="govuk-grid-row">
    <div class="govuk-grid-column-two-thirds">
    </div>
</div>
```

Reply:
That's what I was looking for!